til-domain/src/test/java/com/til/application/interview/InterviewServiceTest.java

```diff
@@ -43,18 +51,35 @@ public class InterviewServiceTest {
         given(interviewRepository.existsByCode(anyString())).willReturn(true);
 
         // when & then
-        assertThatThrownBy(() -> interviewService.create(createInterviewCreateDto()))
+        assertThatThrownBy(() -> interviewService.createInterview(createInterviewCreateDto()))
             .isInstanceOf(BaseException.class)
             .extracting(error -> ((BaseException) error).getErrorCode())
             .isEqualTo(InterviewErrorCode.FAIL_CREATE_INTERVIEW);
+    }
 
+    @Test
+    void 모의면접_생성시_이미_진행중인_모의면접이_존재하면_예외를_던진다() {
+        // given
+        given(interviewRepository.existsByUserIdAndStatus(anyLong(), any())).willReturn(true);
+
+        // when & then
+        assertThatThrownBy(() -> interviewService.createInterview(createInterviewCreateDto()))
+            .isInstanceOf(BaseException.class)
+            .extracting(error -> ((BaseException) error).getErrorCode())
+            .isEqualTo(InterviewErrorCode.ALREADY_PROCESSING_INTERVIEW);
     }
 
     private InterviewCreateDto createInterviewCreateDto() {
         return InterviewCreateDto.builder()
             .status(InterviewStatus.PROCESSING)
             .code(RandomValueGenerator.generateRandomId(11))
             .userId(1L)
+            .categoryIdList(new ArrayList<>() {
+
+                {
+                    add(1L);
+                }
+            })
```

Comment:
P4: 아래처럼 리팩토링 해보면 좋을 것 같습니다!
```suggestion
            .categoryIdList(List.of(1L)) 
```

Reply:
반영했습니다!